parser.js

```diff
@@ -36,7 +36,10 @@ function buildRuleset(name, rules, processors) {
       }
 
       if (value) {
-        return value.trim();
+        if (value.trim) {
+          return value.trim();
+        }
+        return value;
```

Comment:
This was required because the keywords post-processor converts the String to an Array, which doesn't have a `.trim()` method, so it was blowing up.